### PR TITLE
GROWTH-6266 Allow only the exact /soa/graphql/ endpoint

### PR DESCRIPTION
**Pull request by nameofname**
### IMPORTANT

Commits to robots.txt landing in master will be pushed to production automatically. Only merge pull requests that you're confident should be pushed to production.

### CODE REVIEW

**Review by robrichard — APPROVED** [review]

**Review by twhid — APPROVED** [review]

File: www.1stdibs.com/robots.txt [hunk 1]
```diff
@@ -1,8 +1,8 @@
 User-agent: *
-Allow: /soa/graphql/
+Allow: /soa/graphql/$
```
Comment (tml428) [inline www.1stdibs.com/robots.txt:2]: would this potentially bring  back the original issue where some pages can't get rendered by google bot?

Comment (nameofname) [reply www.1stdibs.com/robots.txt:2]: @tml428 the point of the `$` is to specify the end-of-line, so /soa/graphql and /soa/graphql/ will be allowed for the bot.

Comment (tml428) [reply www.1stdibs.com/robots.txt:2]: 👍 👍  I thought there were params following /graphql/ in the requests, which is not the actual case 🤦‍♀️

**Review by tml428 — APPROVED** [review]

File: www.1stdibs.com/robots.txt (hunk 1 above)
Comment (robrichard) [reply www.1stdibs.com/robots.txt:2]: That's a good point about query parameters, maybe it should be something like?

```
Allow: /soa/graphql/$
Allow: /soa/graphql/?*
Allow: /soa/graphql$
Allow: /soa/graphql?*
```

Comment (nameofname) [reply www.1stdibs.com/robots.txt:2]: @robrichard @tml428 From my spot check on major page types we aren't adding any query params right now, but if we did, those URLs would be disallowed given the current robots setup. 

I already released this change and it seems to be working fine, but it's super easy to add the extra allow rules to cover everything... in the case that we _might_ want to add params to the URL in the future. I'm opening a new PR for that and we can discuss there.

**Pull request merged**